### DISCUSSION TITLE
SPL MetaData Phantom incorrect

### DISCUSSION THREAD

**Discussion by LoafCatSolana** [phantom/docs#145] [Help & Support] [answered]
### Summary

During the launch of LoafCat the metadata was different to prevent people discovering. After the launch the MetaData has been updated and reflected in solscan. But on the phantom wallet its still wrong.

Here is the token metadata:
https://solscan.io/token/3de2yRhtD4VbJBb8EQAQffYMPLU4EnSHT1eveBwiL3tn#metadata

Can you force refresh the data in a way?

### Steps to Reproduce

Add the token to Phantom, and it shows up as BOW.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by LoafCatSolana** [comment]
@adamdelphantom , i saw you responding in some similar issue. Is there a way you can help us as well?

**Reply by adamdelphantom** [reply, marked as answer]
Hi @LoafCatSolana,

This image appears to be updated in Phantom. Please let me know if you have any other issues.
